### ISSUE TITLE
readme suggests lzma,9

### ISSUE THREAD

**Issue by ThomasWaldmann** [michabbs/proxborg#2]
... and that is a really bad idea.

From changelog: "make clear that lzma levels > 6 are a waste of cpu cycles"

I didn't find the change in the actual docs, maybe you can search again.

The reason for this is that borg does only feed a few MBs of data at once into lzma and the higher (>6) lzma levels would need more data to get better than at level 6.

Besides that, I guess I would not suggest lzma anyway except if you want to produce a lot of heat.

zstd is way better, so rather use / recommend that.

**Comment by michabbs** [comment]
The "lzma" recomendation affects only the "first" backup. The point is to make the repository as small as possible on the very first run, because this way all deduplicatable data will stay in repository forever with high compression ratio. Benchmarks which I found show that lzma is still better (in terms of compession ratio) than zstd. That's why I recommended lzma, even through it would take plenty of time (which is also mentioned in the README).
